doc/Comments.xml

```diff
@@ -408,21 +408,29 @@ not finished or temporary files.
 
 <Subsection Label="@BeginChunk">
 <Index Key="@BeginChunk"><C>@BeginChunk, @EndChunk, and @InsertChunk</C></Index>
-<Heading>@BeginChunk <A>name</A>, @EndChunk, and @InsertChunk <A>name</A></Heading>
-Text insider of a @BeginChunk / @EndChunk part will not be inserted into
+<Heading>@BeginChunk <A>name[ param1,...]</A>, @EndChunk, and @InsertChunk <A>name[ param1_value,...]</A></Heading>
+Text inside of a @BeginChunk / @EndChunk part will not be inserted into
 the final documentation directly. Instead, the text is stored in an internal buffer.
 
 That chunk of text can then later on be inserted in any other place by using the
 @InsertChunk <A>name</A> command.
 
+In the text, you can use <A>%param1</A> placeholders, which will then be overwritten
+by the <A>param1_value</A> parameters when the text is inserted. Different parameters
+may not contain spaces and must be separated by a comma. Different values may contain
+spaces and must be separated by a comma. There must be a value for each parameter. Please
+note that this is not checked by &AutoDoc;.
```

Comment:
Why not?

Reply:
I decided not to do this. I can only check this at the point of writing out the xml, as the insertion might be read before the actual chunk. Then I could not point to the place where it is wrong. I can still provide an error message, but it might not contain all information people want. Will insert it anyway.